modules/vpc/variables.tf

```diff
@@ -22,6 +22,7 @@ variable "networks" {
       ip_cidr_range   = "192.168.1.0/24"
       allowed_sources = ["209.85.152.0/22"]
       log_metadata    = "INCLUDE_ALL_METADATA"
+      mtu             = 1500
```

Comment:
should this be 1500 or 1460? I thought the default was 1460.

Reply:
When the parameter is omitted, result will be 1460, current provider default. Although, internally in terraform it's represented a bit different - with '0', this can be observed in network info output. If one specifies `1460` explicitly, it causes network replacement as the 'internal' value changes:

```
  # module.vpc.google_compute_network.this["vcp-test"] must be replaced
-/+ resource "google_compute_network" "this" {
      [...]
      ~ id                              = "projects/<snip>/global/networks/vcp-test" -> (known after apply)
      [...]
      ~ mtu                             = 0 -> 1460 # forces replacement
      [...]
}
```

This variable's description shows shows different scenarios: minimal configuration, more advanced one (with some optional params set to non-default values) and a full one. Actually, similar line could also be added to the last case as well.